### ISSUE TITLE
Pkg.build("Blosc") fails

### ISSUE THREAD

**Issue by finmod** [stevengj/Blosc.jl#31]
Any idea why this .build("Blosc") fails? This is on fresh install of Bash on Ubuntu 16.04 with Julia v0.5:

julia> Pkg.update()
INFO: Updating METADATA...
INFO: Updating cache of DataFrames...
INFO: Updating cache of DataFrames...
INFO: Computing changes...
INFO: Installing Blosc v0.1.7
INFO: Building Blosc
Compiling libblosc...
================================[ ERROR: Blosc ]================================

LoadError: failed process: Process(`make -f ../../make.blosc LIB=../../libblosc.so`, ProcessExited(127)) [127]
while loading /root/.julia/v0.5/Blosc/deps/build.jl, in expression starting on line 21

================================================================================

================================[ BUILD ERRORS ]================================

WARNING: Blosc had build errors.

 - packages with build errors remain installed in /root/.julia/v0.5
 - build the package(s) and all dependencies with `Pkg.build("Blosc")`
 - build a single package by running its `deps/build.jl` script

================================================================================

julia>

**Comment by stevengj** [comment]
Do you have `make` etcetera installed?  What happens if you do:

```
cd /root/.julia/v0.5/Blosc/deps/c-blosc-v0.1.7/blosc
make -f ../../make.blosc LIB=../../libblosc.so
```

**Comment by stevengj** [comment]
(PS. The normal recommendation on Unix is not to build software as `root`.  You generally want to do a little as possible as the root user.  Build software as a normal user, then switch to root, or `sudo`, to install it.)

**Comment by finmod** [comment]
SOLVED. I unexpectedly solve this problem by completing the rest of my Julia setup on Bash on Ubuntu. I installed Juno on bash Ubuntu-Julia and noticed that Blosc and others were being built without flags. I then did a Pkg.upadate() and everything fell into place. I don't know precisely why and I am sure that this makes more sense to you than to me. It should say somewhere in the READ.ME something about the dependencies.
